storage/store.go

```diff
@@ -224,6 +226,14 @@ func newStoreRangeSet(store *Store) *storeRangeSet {
 	}
 }
 
+func newStoreShuffledRangeSet(store *Store) *storeRangeSet {
+	return &storeRangeSet{
+		store:   store,
+		visited: -1,
+		shuffle: true,
+	}
```

Comment:
For future-proofing, better call `newStoreRangeSet` and then set `shuffled: true`.